Sming/SmingCore/Data/CStringArray.h

```diff
@@ -80,16 +80,16 @@ class CStringArray : protected String
 	void clear()
 	{
 		invalidate();
-		count_ = 0;
+		stringCount = 0;
 	}
 
 	unsigned count() const
 	{
-		return count_;
+		return stringCount;
 	}
 
 private:
-	unsigned count_ = 0;

```

Comment:
@mikee47 Are there any other places in the code where you succeeded to "smuggle" underscores in the variable names?

Reply:
Wiring/FlashString.h. I've updated the coding style page as suggested to clarify.